### ISSUE TITLE
Realtime GUI/HUD based alchemical reading from crosshair, rather than scanner. Its aim is wonky.

### ISSUE THREAD

**Issue by GreenboxGames** [naelstrof/KoboldKare#156]
The scanner is clunky and kinda out of place, no offense. It seems better suited for a VR controller but is very lacking in First/Third Person style. It would be nice to just have contents appear in the hud, maybe after things are scaled down a bit. As I issued before, GUI is a bit massive across the board.

**Comment by naelstrof** [comment]
Yeah the UI in general needs to be reworked. I'm not sure if I will get to this, I find the scanner a fun mechanic-- though yeah it's pretty clunky and probably doesn't need to exist.

**Comment by GreenboxGames** [comment]
I think if you plan to ever have a VR version, it works quite well as most elements of the HUD/GUI are external and made more tactile interactively... er interactively tactile? I don't think it's that bad actually now that I think about it...

My thinking, totally IMO and no real judgment, right now you have stats up in the corner, you have your money down left, and your stars to the right, in all the hud, even indicators for what buttons to push and when hovering over, that's good help but after a while is in the way. Everything else is either invisible or shows in the external color or shape of things in the form of natural feedback, which is awesome. I feel like data representation should probably be either in the HUD or represented in the world, rather than some in one and some in the other. 

Again, IMHO, it feels a bit whiplash to not have a consistent, similar information representation to the player and I think natural, visual, and external representations of info suit this game best. This game could easily be no HUD and that would make it easier for a VR mod. I'm a big fan of "The Scouter" like device. It would use World GUI objects to show data, clearing space on the HUD. I feel like for QoL mechanics you'd know what to look for and see what was in an open container, but perhaps not so much in a Kobold. The distinction might be good but more work to separate the two, but it would be a good start in qualifying the scanner, it probably should stay as the prototype for data acquisition and even have mod support, making it more of a multi-tool. Dunno. I don't think it's as bad
